shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -123,6 +125,7 @@ @implementation FlutterViewController {
   // Coalescer that filters out superfluous keyboard notifications when the app
   // is being foregrounded.
   fml::scoped_nsobject<FlutterCoalescer> _updateViewportMetrics;
+  fml::scoped_nsobject<UIScrollView> _scrollView;
```

Comment:
All of that information is documented at the site of instantiation.  We don't have a practice of documenting every instance variable, it would just be a duplication of what is said at the instantiation site.

Reply:
I think that documentation is better served here than down below. If you see a _scrollView and don't know where it is, you can go-to-definition to see the reasoning. It's less direct to see where the scoped_nsobject is set and then find the local variable's construction which populates it. 

You can also add one more line to say that UIScrollView specifically has a https://developer.apple.com/documentation/uikit/uiscrollviewdelegate/1619378-scrollviewshouldscrolltotop callback which we game here as a hint for future maintainers.

Reply:
done